conmon-rs/server/src/child_reaper.rs

```diff
@@ -322,6 +327,28 @@ impl ReapableChild {
                 if let Err(e) = Self::write_to_exit_paths(exit_code, &exit_paths).await {
                     error!(pid, "Could not write exit paths: {:#}", e);
                 }
+
+                if !cleanup_cmd_raw.is_empty() {
```

Comment:
1: can this all go in a helper similar to write_to_exit_paths?
2: we may want to do this asynchronously to allow other containers to make forward progress while this does

Reply:
+1